controller/lib/core/blower_pid.cpp

```diff
@@ -98,19 +87,17 @@ void blower_pid_execute(const BlowerSystemState &desired_state,
 
   Pressure cur_pressure = get_patient_pressure();
 
-  Setpoint = desired_state.setpoint_pressure.kPa();
-  Input = cur_pressure.kPa();
-  myPID.Compute();
-
+  float pid_output =
+      myPID.Compute(/*input=*/cur_pressure.kPa(),
+                    /*setpoint=*/desired_state.setpoint_pressure.kPa());
   // If the blower is not enabled, immediately shut down the fan.  But for
   // consistency, we still run the PID iteration above.
-  if (!desired_state.blower_enabled) {
-    Output = 0;
-  }
-  Hal.analogWrite(PwmPin::BLOWER, static_cast<int>(Output));
+  float output =
```

Comment:
analogWrite takes an int, right?  (Can we frob whatever is missing that's allowing this not to be a compile error?)

Reply:
Oops, thanks. To make it a compile error, we need to enable -Wconversion (did you know -Wall does not actually enable anywhere near _all_ warnings? I didn't!) - I guess it's not on by default because it produces lots of other errors in places less useful than this. But I think we ought to turn it on anyway - but I'll do it in a separate PR.